### PR TITLE
Creates first version of Schema

### LINKED ISSUE #9: Define the resolvers for our apollo-server schema
**Is your feature request related to a problem? Please describe.**
It's necessary define how the data is get, in graphql this is made by resolvers.

**Describe the solution you'd like**
I would like to have a resolve to each data defined

**Related**
(#8)

### LINKED ISSUE #10: Define the apollo-server schema
**Is your feature request related to a problem? Please describe.**
Since we have, the nodes, relationships and resolves, we can define our schema.

**Describe the solution you'd like**
I would like to see a complete definition of schema, for this sprint

**Related**
(#8 and #9)

### PR DESCRIPTION
**Pull request by LukeHxH**
# Description

This PR adds a proof of concept for the Apollo schema that will be used in the API. Now, to make requests to the API, you'll need to provide a Github token generated by Github App authentication, as it was implemented by @jadsonluan. This token is passed via **authorization header**, as illustrated bellow.

TOKEN

Now it's possible to get data from logged in user and from Github organizations.

This fixes #9 and fixes #10.

## Type of change

- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## How Has This Been Tested?

You can make `me` and `organization` queries in the Apollo GraphQL Playground.

Examples:

**Me**:
USAGE

**Organization**:
image

## Important

The following changes do not pass on ESLINT, and I didn't understand what's causing this error. I think that's some kind of problem with *context* arrow function, in `src/index.js`, when the ApolloServer is being defined:

```js
const server = new ApolloServer({
  schema,
  context: ({ req }) => {
    const token = req.headers.authorization || '';

    if (!token) {
      throw new Error('An authentication token is required!');
    }

    return {
      token
    };
  },
});
```

And we can't get data from team members yet, but this will be solved on next updates.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by fanny** [comment]
Uoooooooooou, congratulations for your work, it seems very very good.

But i have some suggestions, following the patterns saw in P2, and of the some discussions, it's better have a folder structure by feature, something like, `/organization`, `/user` etc,

I am based on this links:
- [GraphQL modules feature based graphql modules at scale](https://medium.com/the-guild/graphql-modules-feature-based-graphql-modules-at-scale-2d7b2b0da6da)
- [Spectrum apollo-server chat discussion](https://spectrum.chat/graphql/general/recommendations-for-scale-able-graphql-folder-structure-nodejs~c3936202-f2df-47cc-af96-1d829d34f1d3)

**Comment by LukeHxH** [comment]
@fanny Thanks for the review :heart:.  I'll be doing the changes tomorrow! :smile:

File: src/resolvers/githubDataFetcher.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const fetch = require('node-fetch');
```
Comment (jadsonluan) [inline src/resolvers/githubDataFetcher.js]: Update to [ES6 modules import](http://exploringjs.com/es6/ch_modules.html). 

**tl;dr**
Instead of using require, use import

Comment (LukeHxH) [reply src/resolvers/githubDataFetcher.js]: Feito

**Comment by LukeHxH** [comment]
@fanny and @jadsonluan I finished my work on your requested changes. :smile:
